pandas/tests/series/methods/test_inplace_ops.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from pandas import Series
+import pandas._testing as tm
+
+
+@pytest.mark.parametrize(
+    "dtype1, dtype2, dtype_expected, dtype_mul",
+    (
+        ["Int64"] * 4,
+        ["float"] * 4,
+        ["Int64"] + ["float"] * 3,
+        pytest.param(
+            "Int64",
+            "Float64",
+            "Float64",
+            "Float64",
+            marks=pytest.mark.xfail(reason="Not implemented yet"),
+        ),
+    ),
+)
+def test_series_inplace_ops(dtype1, dtype2, dtype_expected, dtype_mul):
+
```

Comment:
GH ref to either this PR or original issue

Reply:
Done